tests/ui/consts/const-try.stderr

```diff
@@ -2,7 +2,7 @@ error: const `impl` for trait `FromResidual` which is not marked with `#[const_t
   --> $DIR/const-try.rs:15:12
    |
 LL | impl const FromResidual<Error> for TryMe {
-   |            ^^^^^^^^^^^^^^^^^^^
+   |            ^^^^^^^^^^^^^^^^^^^ not marked with `#[const_trait]`
```

Comment:
I'm not certain that this note adds much value. It's just restating what the primary message is saying.

If we were to add another note, I'd prefer if it says something that deepens the intuition here. Like saying "this trait isn't const".